pages/src/source/tutorials/schematics.md

```diff
@@ -304,7 +317,7 @@ You should always apply a [groundlevel tag](../../source/items/tagtool) when mak
 | {% building_link builder %}       |                                                                                                                            | 1 rack per level                                                                                                                           |
 | {% building_link combatacademy %} | 1 combat dummy per level (a pumpkin on top of a bale of hay); 1 bed per level                                              |                                                                                                                                            |
 | {% building_link composter %}     | 1 [compost barrel](../../source/items/compostbarrel) per level                                                             |                                                                                                                                            |
-| {% building_link concretemixer %} | 3 blocks of flowing water with solid blocks below and air blocks above                                                     |                                                                                                                                            |
+| {% building_link concretemixer %} | 3 blocks of flowing water with solid blocks below and air blocks above, and the worker can stand next to                   |                                                                                                                                            |
```

Comment:
"can stand next to ... it?"

Reply:
That works too, but this was my intended phrasing.  Implied objects are still parsable 😁

Reply:
I'd use the word `which` instead of `and` in that case. Rolls better off the tongue.

"3 blocks of flowing water with solid blocks below and air blocks above, which the worker can stand next to"